ips-classes-S2.txt

```diff
@@ -1,3 +1,6 @@
+# k8s cluster
```

Comment:
1. The better name here would be `Application cluster` so as not to leak the implementation details.
2. This file is for classes, not IP addresses
3. Let's wait for Nandor before merging; he may have a different idea of where to put this information.